sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/TranslatingPullSubscriber.java

```diff
@@ -17,12 +17,28 @@
  */
 package org.apache.beam.sdk.io.gcp.pubsublite;
 
+import com.google.cloud.pubsublite.internal.PullSubscriber;
 import com.google.cloud.pubsublite.proto.SequencedMessage;
 import io.grpc.StatusException;
 import java.util.List;
+import java.util.stream.Collectors;
 
-/** A PullSubscriber exposes a "pull" mechanism for retrieving messages. */
-interface PullSubscriber extends AutoCloseable {
-  /** Pull currently available messages from this subscriber. Does not block. */
-  List<SequencedMessage> pull() throws StatusException;
+class TranslatingPullSubscriber implements PullSubscriber<SequencedMessage> {
```

Comment:
Please add some javadoc for this class. Other than that, changes look good to me.

Reply:
Fixed.